src/forking-store.js

```diff
@@ -11,15 +11,21 @@ import {
 const BASE_GRAPH_STRING = "http://mu.semte.ch/libraries/rdf-store";
 
 export default class ForkingStore {
-  graph = graph();
+  internalStore = new Store();
+
+  /**
+   * @deprecated Use `internalStore` instead. Will be removed in the next major release.
```

Comment:
Does this even need to be public at all? If we're not using it in apps I would make internalStore actually private. In which case we would keep this deprecation message but remove the internalStore reference?

Reply:
ah no, this description still needs a small update after making it private.